src/data/json.js

```diff
@@ -1,10 +1,13 @@
+import {Observable} from 'rxjs';
 import person from './person';
 
 class WebexTeamsAPI {
   constructor() {}
 
-  async getPerson() {
-     return person;
+  getPerson() {
+    return new Observable(observer => {
```

Comment:
The observable should be part of the adapter, not the data source.

Reply:
why? whats wrong with having observable in both?